### PR TITLE
874 show shipping restrictions to awaiting page

### PR DESCRIPTION
**Pull request by davidsmejia**
## Issue Number

#874 

## Purpose/Implementation Notes

* adds the info card for shipping restrictions where the user fills out payment information

## Types of changes

- New feature (non-breaking change which adds functionality)

## Functional tests

n/a

## Checklist

- [X] Lint and unit tests pass locally with my changes

## Screenshots

n/a

### CODE REVIEW

**Review by dvenprasad — APPROVED** [review]
🚢 🚢 🚢 

Screen Shot 2021-06-07 at 9 49 17 AM

**Review by kurtwheeler — APPROVED** [review]
Idk if you were waiting on my review or not but this also LGTM!

**Pull request merged**
